docker/build/cicd.centos.Dockerfile

```diff
@@ -27,3 +27,4 @@ RUN rm -rf $GOPATH/src/github.com/algorand/go-algorand && \
     mkdir -p $GOPATH/src/github.com/algorand/go-algorand
 RUN echo "vm.max_map_count = 262144" >> /etc/sysctl.conf
 CMD ["/bin/bash"]
+
```

Comment:
I like to end the file with a blank line since it makes the output of utils like `cat` more readable.